google/cloud/forseti/scanner/scanners/instance_network_interface_scanner.py

```diff
@@ -102,9 +101,8 @@ def _get_project_policies(self):
         project_policies = (
             project_dao
             .ProjectDao(self.global_configs)
-            .get_project_policies('projects',
-                                  self.
-                                  snapshot_timestamp))
+            .get_project_policies('projects')
```

Comment:
This scanner is still referencing `project_dao` located at `google/cloud/forseti/common/data_access` and I believe the file has been removed already.

Reply:
I punt to @blueandgold to advise on this.

Reply:
Good catch guys.  This whole _get_project_policies() is unused.  :)

Reply:
I've opened: https://github.com/GoogleCloudPlatform/forseti-security/issues/1184